src/scores/categorical/contingency_impl.py

```diff
@@ -90,11 +90,24 @@ def get_table(self):
         """
         return self.xr_table
 
-    def accuracy(self):
+    def accuracy(self) -> XarrayLike:
```

Comment:
Have we tested that this works with xr.Datasets or just Datarrays?
If the former, we need to update unit tests, if the latter, this needs to change to -> xr.Dataarray

Reply:
I'll change it to DataArray for now since that's what's tested. I was leaving the door open to the future here, but I'll be more specific.